### DISCUSSION TITLE
Shopware files no longer included in the docker images?

### DISCUSSION THREAD

**Discussion by victor-vanherpt** [shyim/shopware#84] [Q&A] [answered]
Hi @shyim,
First of all, many thanks for providing such great docker image for Shopware :)
I have the understanding that the builds are somehow broken. Before, I had all of the Shopware files within the /var/www/html directory, but now I see in the specific Dockerfiles (i.e. shopware/8.0/6.4.14.0/Dockerfile) that the zip is downloaded and unzipped, but I have the impression somehow the files are not there after the unzip command and others are run.
Could this be failing?
Am I missing something?

Example: https://hub.docker.com/layers/shyim/shopware/6.4.14.0-php8.0/images/sha256-65c07af9f3792036dab0574a8d73afeb465a3cf53aea1cc3738fe4e5aaa70f86?context=explore
The image size does not match the size it should be after having the zip file uncompressed:
<img width="515" alt="imatge" src="https://user-images.githubusercontent.com/531901/232057975-e275aa6b-0027-4a18-92b4-aa004a936216.png">
<img width="886" alt="imatge" src="https://user-images.githubusercontent.com/531901/232058371-fb006ead-d5bb-4c9f-87ca-851e8dae7579.png">

**Comment by shyim** [comment]
I am currently looking why the folder is empty wtf

**Reply by shyim** [reply, marked as answer]
working again with https://github.com/shyim/shopware/actions/runs/4700634998/jobs/8335599787
